### DISCUSSION TITLE
[Projects beta][Critical] Can't select existing repository within an organisation

### DISCUSSION THREAD

**Discussion by sche** [community/community#7547] [Projects and Issues]
We have a lot of repos with the name `safe-smth` (like safe-ios, safe-android, etc.), and we have just a `safe` repo.
Unfortunately, I cannot select the `safe` repository as it never appears in the suggestion list.

![image](https://user-images.githubusercontent.com/2284385/141147121-36a471fd-e263-4510-be12-000118aedf44.png)

![image](https://user-images.githubusercontent.com/2284385/141147224-ea0aa9a8-2b17-4060-9c42-0e1bc8cace4c.png)

**Comment by glorialangreo** [comment]
We are having the exact same issue, and we are not able to work. Please let us know if we can help debuging!

**Comment by azenMatt** [comment]
Hi - thanks for reporting this, we are working on a fix :)

**Comment by acidtone** [comment]
I'm also having this issue with a private repo. 

Is there a timeline to fix this or a workaround? There's no point in using Beta if we can't add Issues from the current repo.

**Reply by azenMatt** [reply]
Hi @acidtone - A work around would be to add issues directly by copy and pasting the URL. 

If you have one issue in the project from a repo, in some cases that repo should now also be in the initial list of options which might make it easier for adding the next issue.

**Comment by paulozoom** [comment]
This issue isn’t limited to adding an issue to a project, but also when converting a draft issue to issue. The repo I want to add to isn’t on the list and doesn't come up in the search, even searching by the exact name.

Every issue in the project is from _that_ same repo, so what @azenMatt has mentioned — “in some cases that repo should now also be in the initial list of options which might make it easier for adding the next issue” — isn’t helping here either.

**Comment by nestor-sk** [comment]
Adding issues has a "reasonable" workaround, but creating issues from a draft is a blocker. Drafts have been a game-changer in the way we work, but ATM we need to use a placeholder repo to create the issue from and then move the issue to the repo we really wanted the issue in. And I have to explain the trick to everybody going through it 😞

**Reply by azenMatt** [reply]
Thanks for your patience with this - we are still actively working on a fix

**Comment by gbro3n** [comment]
I came across this issue because my project was not selectable as described above. I've since found that (in my case) this was because issues was not enabled in settings for the repository in question.

Ref: https://stackoverflow.com/questions/29692626/why-is-issues-tab-not-present-in-my-repos-github-page

**Reply by briankelleher** [reply]
Just to follow up on this for others who bang their head against the wall - if the repository is a forked repository and the original repository does not have issues enabled, you cannot use it in projects, despite the UI allowing you to create issues and link projects in that repository

**Reply by jamaluddinrumi** [reply]
it works

**Reply by mhvelplund** [reply]
When I read this, I screamed inside so loud it came out as weird sad wookie sound that made my coworkers look 🤦‍♀️
I turned off the issues because we use another system for that and spent 30 minutes scratching my head.

**Reply by bondansebastian** [reply]
Saved me from banging my head, I'm on the verge of doing it
